hadoop-ozone/dist/src/main/smoketest/test.sh

```diff
@@ -151,6 +151,8 @@ if [ "$RUN_ALL" = true ]; then
    execute_tests ozone-hdfs "${TESTS[@]}"
    TESTS=("s3")
    execute_tests ozones3 "${TESTS[@]}"
+   TESTS=("security")
```

Comment:
Can we have a few separate security tests
1. security_kerberos
2. security_kerberos_delegationtoken
3. security_kerberos_delegationtoken_blocktoken

Reply:
Do you mean separate robot test files or separate suite as well?

Reply:
Yes, Let's separate robot tests based on this one in follow up JIRAs.